pkg/storage/sqlite/sqlite.go

```diff
@@ -318,3 +316,41 @@ func (s SQLite) handleStatement(ctx context.Context, stmt sqlite.Statement) (sql
 
 	return result, tx.Commit()
 }
+
+// GetQualityProfileItem gets a quality item that belongs to a profile
+func (s SQLite) GetDownloadClient(ctx context.Context, id int64) (model.DownloadClient, error) {
+	stmt := table.DownloadClient.SELECT(table.DownloadClient.AllColumns).FROM(table.DownloadClient).WHERE(table.DownloadClient.ID.EQ(sqlite.Int64(id)))
+	var result model.DownloadClient
+	err := stmt.QueryContext(ctx, s.db, &result)
+	return result, err
+}
+
+// ListQualityProfileItem lists all quality definitions
+func (s SQLite) ListDownloadClients(ctx context.Context) ([]*model.DownloadClient, error) {
+	items := make([]*model.DownloadClient, 0)
+	stmt := table.Indexer.SELECT(table.DownloadClient.AllColumns).FROM(table.DownloadClient).ORDER_BY(table.DownloadClient.ID.ASC())
+	err := stmt.QueryContext(ctx, s.db, &items)
+	return items, err
+}
+
+// DeleteQualityDefinition deletes a quality
+func (s SQLite) DeleteDownloadClient(ctx context.Context, id int64) error {
+	stmt := table.DownloadClient.DELETE().WHERE(table.DownloadClient.ID.EQ(sqlite.Int64(id))).RETURNING(table.DownloadClient.AllColumns)
```

Comment:
Does this need a returning clause if we don't return anything? I think I did that somewhere too and had questioned it

Reply:
I don't think its needed